docs/config-file.md

```diff
@@ -171,6 +171,14 @@ and there are plenty of configuration settings for the indiviual formatters/lint
   By default the formatters and linters run on code files in `plugins/`, `tests/unit/`, and on `noxfile.py`.
   If you have other scripts in your collection that should be checked, you can add them with this option.
 
+* `ruff_config: str | os.PathLike | None` (default `None`):
+  Allows to specify a config file for `ruff`.
+  Use a relative path to `noxfile.py`.
```

Comment:
I see this is the phrasing throughout the doc so I won't make a suggestion on this instance. Although I think it could be a little clearer.

Is this saying that the value should be the location of a ruff configuration that is relative to the project noxfile.py?

Reply:
Yes.

Maybe
```suggestion
  Use a path relative to `noxfile.py`.
```
is already better?